src/main/java/ch/njol/skript/expressions/ExprTarget.java

```diff
@@ -61,23 +61,28 @@
 	"delete targeted entity of player # for players it will delete the target",
 	"delete target of last spawned zombie # for entities it will make them target-less"
 })
-@Since("1.4.2, 2.7 (Reset)")
+@RequiredPlugins("Paper 1.19+ (enhanced raytracing)")
+@Since("1.4.2, 2.7 (Reset), INSERT VERSION (ignore blocks)")
 public class ExprTarget extends PropertyExpression<LivingEntity, Entity> {
 
+	private static final boolean PAPER_RAYTRACE = Skript.methodExists(LivingEntity.class, "rayTraceEntities", int.class, boolean.class);
+
 	static {
 		Skript.registerExpression(ExprTarget.class, Entity.class, ExpressionType.PROPERTY,
-				"[the] target[[ed] %-*entitydata%] [of %livingentities%]",
-				"%livingentities%'[s] target[[ed] %-*entitydata%]");
+				"[the] target[[ed] %-*entitydata%] [of %livingentities%]" + (PAPER_RAYTRACE ? " [ignoring :blocks]" : ""),
+				"%livingentities%'[s] target[[ed] %-*entitydata%]" + (PAPER_RAYTRACE ? " [ignoring :blocks]" : ""));
 	}
 
 	@Nullable
 	private EntityData<?> type;
+	private static boolean ignoreBlocks;
 
 	@Override
 	@SuppressWarnings("unchecked")
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parser) {
 		type = exprs[matchedPattern] == null ? null : (EntityData<?>) exprs[matchedPattern].getSingle(null);
 		setExpr((Expression<? extends LivingEntity>) exprs[1 - matchedPattern]);
+		ignoreBlocks = parser.hasTag("blocks");
```

Comment:
You're checking for `blocks` tag when the tag would be `ignoring `

Reply:
I made it blocks because it represents what it's ignoring better

Reply:
Right. But that's not what the pattern has the parse tag as